lib-opensource/emqx_management/src/emqx_mgmt_cli.erl

```diff
@@ -550,10 +550,22 @@ listeners(["stop", _Proto, ListenOn]) ->
     end,
     stop_listener(emqx_listeners:find_by_listen_on(ListenOn1), ListenOn1);
 
+listeners(["restart", Identifier]) ->
+    case emqx_mgmt:restart_listener(emqx_listeners:find_by_id(Identifier)) of
+        ok ->
+            emqx_ctl:print("Restarted ~s listener successfully.~n", [Identifier]);
+        %% "ws" returns differently
```

Comment:
delete this line.

Reply:
I understood this as "the commend line 557".
If you mean the case clause `{ok, _}`, I don't really understand why, but it seems to come from `emqx_listeners.erl` `start_listener` spec for tcp or http.

Reply:
I meant the commented line at 557.

but now I spotted another issue. will write another comment.